main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/go-playground/webhooks.v5/github"
```

Comment:
Is this the best package to use?

Reply:
Not sure, this package uses the github and payload go packages. https://godoc.org/gopkg.in/go-playground/webhooks.v5 
I chose it because it was simplest to use, this does not mean it's the best choice.

Reply:
This looks fine for now — hopefully we don’t need to change later